fala/apps/laalaa/api.py

```diff
@@ -15,8 +15,11 @@
 
 
 def get_categories():
-    categories = LaalaaProviderCategoriesApiClient.singleton(settings.LAALAA_API_HOST, _).get_categories()
-    return [item for item in sorted(categories.items())]
+    if settings.LAALAA_API_HOST:
+        categories = LaalaaProviderCategoriesApiClient.singleton(settings.LAALAA_API_HOST, _).get_categories()
+        return [item for item in sorted(categories.items())]
+
+    return []
```

Comment:
🙋‍♂ Might worth to note that if in the future, if I would accidentally misconfigure `LAALAA_API_HOST` on an environment, we would have a no categories.

What do you think about making sure we always require a valid setting on production-like environments?

It seems it could be achieved by adding the environment variable to this list:

https://github.com/ministryofjustice/fala/blob/30ea4cdd5acd9709bd6ea6c8345e73e1a041d5bf/fala/settings/production.py#L4